llvm/include/llvm/IR/DebugInfo.h

```diff
@@ -44,6 +44,12 @@ class Module;
 LLVM_ABI TinyPtrVector<DbgDeclareInst *> findDbgDeclares(Value *V);
 /// As above, for DVRDeclares.
 LLVM_ABI TinyPtrVector<DbgVariableRecord *> findDVRDeclares(Value *V);
+/// Finds dbg.coroframe_entry intrinsics declaring local variables as living in
+/// the memory that 'V' points to.
+LLVM_ABI TinyPtrVector<DbgCoroFrameEntryInst *>
+findDbgCoroFrameEntrys(Value *V);
+/// As above, for DVRCoroFrameEntrys.
+LLVM_ABI TinyPtrVector<DbgVariableRecord *> findDVRCoroFrameEntrys(Value *V);
```

Comment:
We should (upstream) replace this with one API that lets you pass a `kind` parameter.